packages/api/src/router/testSupport/prepareUser.ts

```diff
@@ -95,7 +95,7 @@ const generateEmailAddress = (personaName: keyof typeof personas) => {
     // All users use the "test@thenational.academy" mailbox with a `+` alias
     "test",
     // Replace non-alphanumeric characters with -
-    branch.replace(/\W+/g, "-"),
+    branch.replace(/\W+/g, "-").slice(-15),
```

Comment:
@codeincontext - bit of a crude solution here. Will this cause uniqueness issues?

Reply:
Let's go for it and see what happens. I think it'll be fine